mkosi

```diff
@@ -3270,19 +3270,21 @@ def secure_boot_sign(args: CommandLineArguments, root: str, do_run_build_script:
                 os.rename(p + ".signed", p)
 
 
-def xz_output(args: CommandLineArguments, raw: Optional[BinaryIO]) -> Optional[BinaryIO]:
+def comp_output(args: CommandLineArguments, raw: Optional[BinaryIO]) -> Optional[BinaryIO]:
```

Comment:
I think naming this more verbosely `compress_output` would be clearer.

Reply:
Damn that was the first name I used, but decided to commit a shorter one.

Reply:
fixed